apprentice_teacher.rb

```diff
@@ -1,22 +1,24 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
-  attr_accessor :name
+require_relative 'senior_teacher'
+
```

Comment:
A Student is not an ApprenticeTeacher.   Use the IS A to check the inheritance relationship.  

What we are looking for here is:

- An ApprenticeTeacher IS A Teacher
- A SeniorTeacher IS A Teacher
- A Teacher IS A Person
- A Student IS A Person
- a Person includes the HighFive behavior.

How could you rework this like that?